### DISCUSSION TITLE
Importing unsupported third-party Python libraries

### DISCUSSION THREAD

**Discussion by qziov** [pyscript/pyscript#1936] [Q&A]
For example, when referring to the library flopy in the configuration file, an error will be reported. It may be a third-party library that pyscript cannot support. How can I correctly refer to it
configuration file
<img width="298" alt="1" src="https://github.com/pyscript/pyscript/assets/103866547/5237878e-3f0d-4bd6-9a38-b993f83a7790">
error
<img width="742" alt="2" src="https://github.com/pyscript/pyscript/assets/103866547/2a21c921-4880-4b58-bfb7-55af83b8e76f">
The explanation I found is that Pandas is a powerful Python library for data analysis and processing, but it has some dependencies on the C language, so it cannot be installed directly with microip.

**Comment by WebReflection** [comment]
I am pretty sure we have pandas examples and demos out there https://pyscript.com/@examples/pandas/latest
